drivers/atwinc15x0/atwinc15x0_netdev.c

```diff
@@ -645,7 +645,7 @@ static int _atwinc15x0_get(netdev_t *netdev, netopt_t opt, void *val,
     assert(dev);
     assert(dev == atwinc15x0);
 
-    DEBUG("%s dev=%p opt=%u val=%p max_len=%u\n", __func__,
+    DEBUG("%s dev=%p opt=%u val=%p max_len=%" PRIuSIZE "\n", __func__,
```

Comment:
looks like those macros are not known here?

Reply:
It's missing `#include "architecture.h"`.
The header is included in many places, so it tried to get away with not including it in every file that uses `PRIuSIZE`. I am currently building all examples/tests with the board to see if there are other problems with this board.

If it's ok, I'll add the missing includes and force push a version with the missing includes.